### PR TITLE
Added: alt attribute to few images

### PR DESCRIPTION
**Pull request by chaitanyaj2222**
#### Summary

* I have added alt attributes to a handful of items in the content
#### Motivation

* Adding the alt attribute to the image displays the value of `alt` attribute when the respective `image fails to load` rather than leaving the space blank. It is also read aloud by the screen readers when there is an image to describe the user about the image.
#### Supporting details

@estelle  has collected the list of images from the documentation that do not have the alt attribute. [link to the spreadsheet](https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0)
#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Chaitanya7666** [comment]
#19334

File: files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 1]
```diff
@@ -185,12 +185,12 @@ Then select the _All authors_ link, and then select a particular author. Finally
 If the author has no books, you'll be presented with a page like this.
 After pressing delete, the server will delete the author and redirect to the author list.
 
-![](locallibary_express_author_delete_nobooks.png)
+![The Delete Author section of the Local library application of an author who does not have any books. The left column has a vertical navigation bar. The right section contains the author details with a heading that contains the Author's name followed by the life dates of the author. There is a button labelled 'Delete' at the bottom.](locallibary_express_author_delete_nobooks.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md:188]:
```suggestion
![The Delete Author section of the Local library application of an author who does not have any books. The left column has a vertical navigation bar. The right section contains the author's name and life dates. There is the question "Do you really want to delete this author" with a button labeled 'Delete'.](locallibary_express_author_delete_nobooks.png)
```

File: files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 2]
```diff
@@ -185,12 +185,12 @@ Then select the _All authors_ link, and then select a particular author. Finally
...
 
 If the author does have books, then you'll be presented with a view like the following.
 You can then delete the books from their detail pages (once that code is implemented!).
 
-![](locallibary_express_author_delete_withbooks.png)
+![The Delete Author section of the Local library application of an author who does have books under his name. The left column has a vertical navigation bar. The right section contains the author details with a heading that contains the Author's name followed by the life dates of the author. There is a statement that reads "Delete the following books before attempting to delete this author" and the books written by the author are listed after that.](locallibary_express_author_delete_withbooks.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md:193]:
```suggestion
![The Delete Author section of the Local library application of an author who does have books under his name. The section contains the author's name and life dates of the author. There is a statement that reads "Delete the following books before attempting to delete this author" followed by the author's books. The list includes the titles of each book, as links, followed by a brief description in plain text.](locallibary_express_author_delete_withbooks.png)
```

File: files/en-us/learn/server-side/express_nodejs/routes/index.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ As we've already created the models, the main things we'll need to create are:
 - Controller functions to get the requested data from the models, create an HTML page displaying the data, and return it to the user to view in the browser.
 - Views (templates) used by the controllers to render the data.
 
-![](mvc_express.png)
+![Main data flow diagram of an MVC express server: 'Routes' receive the HTTP requests sent to the Express server and forward them to the appropriate 'controller' function. Controller is used to read and write data from the models. Models are connected to the database to provide data access to the server. Controllers use 'views' to render the data and sends it back to the client as HTTP response.](mvc_express.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/express_nodejs/routes/index.md:54]:
```suggestion
![Main data flow diagram of an MVC express server: 'Routes' receive the HTTP requests sent to the Express server and forward them to the appropriate 'controller' function. The controller reads and writes data from the models. Models are connected to the database to provide data access to the server. Controllers use 'views', also called templates, to render the data. The Controller sends the HTML HTTP response back to the client as an HTTP response.](mvc_express.png)
```

**Review by estelle — APPROVED** [review]
fantastic. Thank you!

**Comment by Chaitanya7666** [comment]
Thank you!

**Pull request merged**